staging/src/kubevirt.io/api/migrations/v1alpha1/types.go

```diff
@@ -99,6 +101,14 @@ func (m *MigrationPolicy) GetMigrationConfByPolicy(clusterMigrationConfiguration
 		changed = true
 		*clusterMigrationConfigurations.AllowPostCopy = *policySpec.AllowPostCopy
 	}
+	if policySpec.AllowWorkloadDisruption != nil {
+		changed = true
+		*clusterMigrationConfigurations.AllowWorkloadDisruption = *policySpec.AllowWorkloadDisruption
+	} else if policySpec.AllowWorkloadDisruption == nil && policySpec.AllowPostCopy != nil {
```

Comment:
If you insist :)

I wanted to be explicit because just on line 100 we have another `if policySpec.AllowPostCopy != nil`
I was worried that it may bring more confusion.
If you think it's redundant, I don't mind removing it.

Reply:
That's fine :) 
I think that in general `GetMigrationConfByPolicy` deserves a cleanup (e.g. the returned error is always nil), but it is out of the scope of this PR. Write it here just to share it :slightly_smiling_face: